tests/contrib/operators/test_dataproc_operator_system.py

```diff
@@ -18,16 +18,29 @@
 # under the License.
 import unittest
 
-from tests.contrib.utils.base_gcp_system_test_case import SKIP_TEST_WARNING, TestDagGcpSystem
+from airflow.contrib.example_dags.example_gcp_dataproc import OUTPUT_PATH
+
+from tests.contrib.utils.base_gcp_system_test_case import (
+    SKIP_TEST_WARNING,
+    TestDagGcpSystem,
+)
 from tests.contrib.utils.gcp_authenticator import GCP_DATAPROC_KEY
+from tests.contrib.operators.test_dataproc_operator_system_helper import (
+    DataprocTestHelper,
+)
 
 
 @unittest.skipIf(TestDagGcpSystem.skip_check(GCP_DATAPROC_KEY), SKIP_TEST_WARNING)
 class DataprocPigOperatorExampleDagsTest(TestDagGcpSystem):
     def __init__(self, method_name="runTest"):
         super().__init__(
-            method_name, dag_id="example_gcp_dataproc_pig_operator", gcp_key=GCP_DATAPROC_KEY
+            method_name, dag_id="example_gcp_dataproc", gcp_key=GCP_DATAPROC_KEY
         )
+        self.helper = DataprocTestHelper()
```

Comment:
I think it's worth adding the addition code to create a bucket with content.
Example: https://github.com/apache/airflow/blob/master/tests/gcp/operators/test_cloud_build_system_helper.py#L59-L63